templates/bin/run

```diff
@@ -0,0 +1,6 @@
+#!/bin/zsh
+
+source ~/.zshrc
+
+(sleep 4 && open 'http://<appname>.dev:3000') &
+rails s -b <appname>.dev
```

Comment:
i think it should not make a difference? but yes, i think it's better to call bin/rails, fixing it!